### ISSUE TITLE
Changes needed in HTML and CSS @kroy96

### ISSUE THREAD

**Issue by V-1-V-3-K** [incubateind/gdtc-Day2#80]

**Comment by rexyagami** [comment]
@V-1-V-3-K can you explain the issue, please?

**Comment by V-1-V-3-K** [comment]
1.padding in sections, 
2.aligning the heading in the center, 
3.and labels in the progress bar are incorrect
@rexyagami

**Comment by rexyagami** [comment]
@V-1-V-3-K yes that's correct, you have to change and resolve these design flaws and achieve the final design as shown in the image.
